script/Arcueid/Day 5/05_00_ARC05_5B.txt

```diff
@@ -319,22 +319,22 @@ Am I afraid of confronting the violence... of confronting that monster?
 [sha:a07139bc82621e007d5693d58e7f058aef701988]{
 -- Page 443, Offset 9765.
 -- そうであるなら問題はない。
-If that's the case, then it's fine.
+If that's what it is, then there's no problem.
 }
 [sha:2ba2dcd0ac808834dcdfb7883e22c5d10d95fdec]{
 -- Page 443, Offset 9766.
 -- アレを恐れるのは人間として正常だ。近寄るだけで殺される相手と向き合う。恐怖がなくてまともな人間でいられるものか。
-It's normal for people to be afraid of that thing. Of facing someone that could kill you just by drawing near. How can you call yourself human if you aren't afraid of that?
+It's only natural that a human being would be afraid of that %{i}thing%{/i}. Afraid of confronting a being that you can't even approach without coming face-to-face with death. Could you really call yourself human if you weren't afraid of that?
```

Comment:
Suggestion, might be a little too liberal.
> Afraid of confronting a being that emanates death from his very presence.

Change last sentence to present tense, the thought is occurring now.